targets/targets.json

```diff
@@ -2259,7 +2259,7 @@
         "supported_toolchains": ["ARM", "GCC_ARM", "IAR"],
         "default_toolchain": "ARM",
         "program_cycle_s": 2,
-        "device_has": ["ANALOGIN", "CAN", "ETHERNET", "I2C", "I2CSLAVE", "INTERRUPTIN", "PORTIN", "PORTINOUT", "PORTOUT", "PWMOUT", "RTC", "SERIAL", "SERIAL_FC", "SPI", "SPISLAVE", "STDIO_MESSAGES"],
+        "device_has": ["ANALOGIN", "CAN", "ETHERNET", "I2C", "I2CSLAVE", "I2C_ASYNCH", "INTERRUPTIN", "PORTIN", "PORTINOUT", "PORTOUT", "PWMOUT", "RTC", "SERIAL", "SERIAL_ASYNCH", "SERIAL_FC", "SPI", "SPISLAVE", "SPI_ASYNCH", "STDIO_MESSAGES"],
```

Comment:
Why is this change included?

Reply:
Because it will be needed when the two targets (VK_RZ_A1H and RZ_A1H) are merged look here  #4440 
>... after adding "I2C_ASYNCH", "SPI_ASYNCH" & "SERIAL_ASYNCH" interfaces as "device_has" ittems for the "VK_RZ_A1H" platform in targets.json, the errors disappeared and the test finally managed to run.

Reply:
Right, but that should not be part of *this* PR, and instead part of another one.

Reply:
OK it will be removed then.

Reply:
OK targets.json is revertd ! Happy ?